config/owasp/suppressions.xml

```diff
@@ -1,544 +1,262 @@
-   file name: jetty-repacked-9.4.12.v20180830.jar (shaded: org.eclipse.jetty:jetty-xml:9.4.12.v20180830)
+   file name: tomcat-embed-websocket-9.0.31.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.eclipse\.jetty/jetty-xml@.*$</packageUrl>
-        <cpe>cpe:/a:eclipse:jetty</cpe>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
+        <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: jetty-repacked-9.4.12.v20180830.jar (shaded: org.eclipse.jetty:jetty-xml:9.4.12.v20180830)
+   file name: tomcat-embed-websocket-9.0.31.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.eclipse\.jetty/jetty-xml@.*$</packageUrl>
-        <cpe>cpe:/a:jetty:jetty</cpe>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
+        <cpe>cpe:/a:java-websocket_project:java-websocket</cpe>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: jetty-repacked-9.4.12.v20180830.jar (shaded: org.eclipse.jetty:jetty-xml:9.4.12.v20180830)
+   file name: dom4j-2.1.1.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.eclipse\.jetty/jetty-xml@.*$</packageUrl>
-        <cve>CVE-2019-10241</cve>
-        <cve>CVE-2019-10247</cve>
+        <packageUrl regex="true">^pkg:maven/org\.dom4j/dom4j@.*$</packageUrl>
+        <vulnerabilityName>CVE-2020-10683</vulnerabilityName>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   This suppresses a CVE-2013-4580 cve in any gitlab
+   file name: spring-security-core-5.2.2.RELEASE.jar
    ]]></notes>
-        <cpe>cpe:/a:gitlab:gitlab</cpe>
-        <cve>CVE-2013-4580</cve>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
+        <vulnerabilityName>CVE-2020-5407</vulnerabilityName>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: log4j-to-slf4j-2.12.1.jar
+   file name: tomcat-embed-core-9.0.31.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.logging\.log4j/log4j\-to\-slf4j@.*$</packageUrl>
-        <cve>CVE-2020-9488</cve>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
+        <vulnerabilityName>CVE-2020-9484</vulnerabilityName>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: log4j-api-2.12.1.jar
+   file name: spring-security-core-5.2.2.RELEASE.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.logging\.log4j/log4j\-api@.*$</packageUrl>
-        <cve>CVE-2020-9488</cve>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
+        <vulnerabilityName>CVE-2020-5408</vulnerabilityName>
     </suppress>
-</suppressions>
\ No newline at end of file
+</suppressions>
```

Comment:
A lot of these suppressions seem to have the same file name/packageUrl - therefore they can be placed under one `<suppression>` tag for example:

```
    <suppress>
        <notes><![CDATA[
   file name: postgresql-42.2.9.jar
   ]]></notes>
        <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
        <cpe>cpe:/a:postgresql:postgresql</cpe>
        <cpe>cpe:/a:postgresql:postgresql_jdbc_driver</cpe>
        <cve>CVE-2020-13692</cve>
    </suppress>
```

or for a suppression with multiple CVE's/CPE's use a regex/suppression:

    <suppress>
        <notes><![CDATA[
    file name: TheCommonPartOfTheJarName*.jar
    ]]></notes>
        <filePath regex="true">.*\**\.jar</filePath>
        <cpe>cpe:/TheCPESuppressionForTheJar</cpe>
    </suppress>

Reply:
I think we can upgrade postgres in gradle and remove this suppression ( compile group: 'org.postgresql', name: 'postgresql', version: '42.2.8')